src/StackExchange.Redis/PhysicalBridge.cs

```diff
@@ -884,7 +884,7 @@ private async Task ProcessBacklogAsync()
                             if (maxFlush >= 0) ex.Data["Redis-MaxFlush"] = maxFlush.ToString() + "ms, " + (physical?.MaxFlushBytes ?? -1).ToString();
                             if (_maxLockDuration >= 0) ex.Data["Redis-MaxLockDuration"] = _maxLockDuration;
 #endif
-                            message.SetExceptionAndComplete(ex, this);
+                            message.SetExceptionAndComplete(ex, this, onConnectionRestoreRetry: false);
```

Comment:
Note for potential enum re-use, rather than true/false here. We have a few different ways to describe the retry depending on the point in the stack from config through execution, let's see what we can do to unify.

Reply:
thanks for the feedback here, simplified it in the next iteration